test/augmentation/test_augmentation.py

```diff
@@ -547,27 +538,31 @@ def test_random_contrast_list(self, device, dtype):
                                 [0.7, 0.8, 1.]]]], device=device, dtype=dtype)  # 1 x 1 x 3 x 3
         input = input.repeat(2, 3, 1, 1)  # 2 x 3 x 3
 
-        expected = torch.tensor([[[[0.0849, 0.1699, 0.2548],
-                                   [0.5097, 0.4247, 0.3398],
-                                   [0.5946, 0.6795, 0.8494]],
-                                  [[0.0849, 0.1699, 0.2548],
-                                   [0.5097, 0.4247, 0.3398],
-                                   [0.5946, 0.6795, 0.8494]],
-                                  [[0.0849, 0.1699, 0.2548],
-                                   [0.5097, 0.4247, 0.3398],
-                                   [0.5946, 0.6795, 0.8494]]],
-                                 [[[0.0821, 0.1642, 0.2463],
-                                   [0.4926, 0.4105, 0.3284],
-                                   [0.5747, 0.6568, 0.8210]],
-                                  [[0.0821, 0.1642, 0.2463],
-                                   [0.4926, 0.4105, 0.3284],
-                                   [0.5747, 0.6568, 0.8210]],
-                                  [[0.0821, 0.1642, 0.2463],
-                                   [0.4926, 0.4105, 0.3284],
-                                   [0.5747, 0.6568, 0.8210]]]], device=device, dtype=dtype)
+        expected = self.expected_contrast(device, dtype)
 
         assert_allclose(f(input), expected, atol=1e-4, rtol=1e-5)
 
+    def expected_saturation(self, device, dtype):
```

Comment:
all this tests will change with https://github.com/kornia/kornia/pull/706, right ?

Reply:
We need to update both sides. This PR will give more functionalities. #706 shall have more basic testings, like different device/dtype operations shall have different value outputs, etc.